react/features/filmstrip/functions.web.js

```diff
@@ -94,17 +94,13 @@ export function calculateThumbnailSizeForTileView({
     clientWidth,
     clientHeight
 }: Object) {
-    // The distance from the top and bottom of the screen, as set by CSS, to
-    // avoid overlapping UI elements.
-    const topBottomPadding = 200;
-
     // Minimum space to keep between the sides of the tiles and the sides
     // of the window.
     const sideMargins = 30 * 2;
 
     const verticalMargins = visibleRows * 10;
```

Comment:
Is this *10 not essentially the top and bottom 5px that you changed to 2px?

Reply:
I think that's ok. At any rate we are going to take it for a spin on beta (thus landing it) and iterate through it if we see problems. Thanks for taking a look!